__tests__/server/utils/onModuleLoad.spec.jsx

```diff
@@ -180,6 +184,31 @@ describe('onModuleLoad', () => {
     });
     expect(setStateConfig).toHaveBeenCalledWith(provideStateConfig);
   });
+  it('calls setStateConfig if setRedirectAllowList is supplied', () => {
+    onModuleLoad({
+      module: {
+        [CONFIGURATION_KEY]: {
+          redirectAllowList: ['https://americanexpress.com'],
+          csp,
+        },
+        [META_DATA_KEY]: { version: '1.0.5' },
+      },
+      moduleName: 'some-root',
+    });
+    expect(setRedirectAllowList).toHaveBeenCalledWith(['https://americanexpress.com']);
+  });
+  it('calls setStateConfig with empty array if setRedirectAllowList is NOT supplied', () => {
```

Comment:
```suggestion
  it('calls setRedirectAllowList with empty array if setRedirectAllowList is NOT supplied', () => {
```

Reply:
Fixed in eb2c2281679bf7f5c23c4d2b1c8bfd0f879fd147